src/UrlGenerator/BaseUrlGenerator.php

```diff
@@ -83,4 +83,13 @@ public function rawUrlEncodeFilename(string $path = ''): string
     {
         return pathinfo($path, PATHINFO_DIRNAME).'/'.rawurlencode(pathinfo($path, PATHINFO_BASENAME));
     }
+
+    public function versionUrl(string $path = ''): string
+    {
+        if (! $this->config->get('medialibrary.versioning', false)) {
```

Comment:
I don't think the `false` default is needed, because the config of the project gets merged with the file in the package. I guess all the tests will still be green when you remove the default here.